src/utils.js

```diff
@@ -85,6 +85,22 @@ export default {
 			});
 		});
 	},
+	editBody(data) {
```

Comment:
Possibly make a common method called `editFile` and have both `editLog` and `editBody` use it?